plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestIcebergTrinoRestCatalogConnectorSmokeTest.java

```diff
@@ -70,8 +76,8 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
         return switch (connectorBehavior) {
             case SUPPORTS_CREATE_MATERIALIZED_VIEW,
-                SUPPORTS_RENAME_MATERIALIZED_VIEW,
-                SUPPORTS_RENAME_SCHEMA -> false;
+                 SUPPORTS_RENAME_MATERIALIZED_VIEW,
+                 SUPPORTS_RENAME_SCHEMA -> false;
```

Comment:
Separate a commit. This change is unrelated to "Add multipart namespace support to TrinoRestCatalog".

Reply:
reverted this changes